Gemfile

```diff
@@ -51,6 +51,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
```

Comment:
You did great job! Your test are passing correctly. 
[OPTIONAL] However, kindly remove uncreated pending tests from your file to make your code cleaner.

![Screen Shot 2022-08-25 at 05 46 03](https://user-images.githubusercontent.com/69011963/186549008-d11dd99e-9226-425f-9f50-977f599371f9.png)